model-providers/openai/openai-common/runtime/src/main/java/io/quarkiverse/langchain4j/openai/QuarkusOpenAiClient.java

```diff
@@ -526,6 +545,11 @@ public Builder azureAdToken(String azureAdToken) {
             return this;
         }
 
+        public Builder configName(String modelName) {
```

Comment:
We should probably stick with either `configName` or `modelName`

Reply:
nice catch. I would prefer sticking with configName since modelName is taken by something else during the setup of the ImageModel.